src/main/java/network/brightspots/rcv/StreamingCvrReader.java

```diff
@@ -199,13 +199,16 @@ private void endCvr() {
       encounteredDataErrors = true;
     }
 
+    // Log the raw data for auditing
+    Logger.fine(" [Raw Data] ");
+    Logger.fine(currentCvrData.toString());
```

Comment:
So, as suggested in the filed issue, you've moved the raw data logging so that it all happens when we're first ingesting the CVR rather than being included as part of the round 1 output for each CVR. We should run this by the RCVRC folks to confirm that they're OK with the slight change to the audit log format.

Also worth noting that (I believe) the audit log doesn't contain the raw ingested data at all when the CVR source isn't XLS...

Reply:
I would just email Rene/Chris/George to explain what's changing and see if anyone objects. It's not conforming to any official standard; it's just something we agreed on as a group years ago.

Reply:
Also, just want to confirm: this still goes to the correct file, yes?

Reply:
Yes -- I'll post a before/after audit log to ensure everyone is on board.